src/main/scala/com/lucidchart/open/relate/SqlStatementParser.scala

```diff
@@ -101,15 +94,10 @@ private[relate] object SqlStatementParser {
    * @param query the current query in a StringBuilder
    */
   private def insertTuples(numTuples: Int, tupleSize: Int, query: StringBuilder): Unit = {
-    if (numTuples > 0) {
+    for (i <- 0 to numTuples - 1) {
```

Comment:
@eurafa Not that @msiebert  used _until_ rather than _to_. _until_  is exclusive on the end, whereas _to_ is inclusive on the end.